Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -170,6 +173,16 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
     }
   };
 
+  publicAPI.getContainer = () => model.el;
+
+  publicAPI.getContainerSize = () => {
+    if (!model.containerSize && model.el) {
+      const { width, height } = model.el.getBoundingClientRect();
+      model.containerSize = [width, height];
+    }
+    return model.containerSize || [300, 300];
```

Comment:
is the `[300, 300]` defined anywhere else?

Reply:
Actually, why do we return `[300,300]` when `model.el === undefined` and not just return `undefined`?

Reply:
Because we need to have valid numbers no matter what.

Reply:
Why? `getContainer` can return something `undefined`. If there is no valid container, should we have valid numbers when requesting a size?

Reply:
Using the actual canvas size when no container is provided is even better.

Reply:
Isn't that even more dangerous? We think we get a valid container size, but it's actually a canvas size which is not from the same space.

Reply:
No, in fact it is the perfect way to handle it since you have no container size constraint and will naturally have a 1 to 1 mapping. To me, that makes perfect sense. 
Remember we are dealing with a corner case.

Reply:
okey!